circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  node:
+    version: 5.5.0
```

Comment:
There is not IMO.
This is the higher version that CircleCI can offer right now.